org/mozilla/jss/nss/PR.java

```diff
@@ -88,18 +92,14 @@ public static int Close(PRFDProxy fd) {
      *           org.mozilla.jss.nss.SSLFDProxy.releaseNativeResources
      */
     public synchronized static int Close(SSLFDProxy fd) throws Exception {
```

Comment:
I think that's fine -- its a simplification over `Close(PRFDProxy, boolean)` -- otherwise, callers would have to know that `SSLFDProxy` _has_ to be closed via `Close(fd, false)` and keep that in mind at every call site.

Reply:
If this is a purely object-oriented design, such details should have been handled in `PRFDProxy.close()` and `SSLFDProxy.close()`, but let's not worry about this internal API.